README.md

```diff
@@ -1 +1,168 @@
-# My-Portfolio
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="./img/logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>My Portfolio</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 My Portfolio <a name="about-project"></a>
+
+> This is my Portfolio I created it using HTML, CSS and JavaScript.
+
+## 🛠 Built With <a name="built-with"></a> 
+
+
+1- HTML.
+2- CSS.
+3- Liners.
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">HTML</a></li>
+    <li><a href="https://reactjs.org/">CSS</a></li>
+    <li><a href="https://reactjs.org/">JavaScript</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Added index.html file.
+- Added style.css file.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+1. Web browser.
+2. Code editor.
+3. Git-smc.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Run this command:
+
+```sh
+  cd my-folder
+  git clone https://github.com/pradmon/MyPortfolio.git
+```
+
+### Install
+
+Install this project with:
+
+Run this command:
+
+```sh
+  cd my-project
```

Comment:
- [ ] To make your documentation more professional, I think it would be better if you update tothe the `cloning link` and the `accessed` cloned directory command to the correct ones. Your project is named is named `My-Portfolio` and not `MyPortfolio`. they should be:
```
...
git clone https://github.com/pradmon/My-Portfolio.git
```
```
cd My-Portfolio
...
```

Reply:
Corrected link.